src/debug.module.ts

```diff
@@ -276,14 +276,25 @@ export class DebugModule {
         Type<any>
       >) || [];
     for (const dep of typedDeps) {
-      retDeps.push(dep.name);
+      if (!dep) {
+      }
```

Comment:
```suggestion
```

dead code

Reply:
Thank you. Knew I would miss something. I initially had a log right there, but needed to move it to later in the code